parallel-install/pkg/preinstaller/preinstaller.go

```diff
@@ -25,19 +25,19 @@ package preinstaller
 import (
 	"fmt"
 	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
 	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
 	"io/ioutil"
 	"k8s.io/client-go/dynamic"
-	"k8s.io/client-go/tools/clientcmd"
-
 	"os"
 )
 
 // Config defines configuration values for the PreInstaller.
 type Config struct {
 	InstallationResourcePath string           //Path to the installation resources.
 	Log                      logger.Interface //Logger to be used
-	KubeconfigPath           string
+	KubeconfigPath           string           // Path to the kubeconfig file
```

Comment:
perhaps it's easier to inject `*rest.Config` here instead?

Reply:
For now the trend is to pass all kubeconfigs. I think that library's consumers want to simply pass config properties, such as kubeconfigs.

Reply:
yeah ofc we can do that buuuuuut helm need kubeconfigpath and that's the problem